cmd/hypper/search.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/Masterminds/log-go"
+	"github.com/rancher-sandbox/hypper/pkg/search"
+	"github.com/spf13/cobra"
+)
+
+const searchDesc = `
+Search provides the ability to search for Helm charts in the various places
```

Comment:
```suggestion
Search provides the ability to search for Hypper charts in the various places
```
Just realized..

Reply:
woops, if you accept the suggestion it auto resolves, sorry about that!

Reply:
yep, the missing `Signed-off-by: Víctor Cuadrado Juan <vcuadradojuan@suse.de>` is a pity :/. Can you amend the last commit?

Reply:
thanks! :)